### PR TITLE
Convert interapps-landing into a standalone service

### PR DESCRIPTION
**Pull request by johnworth**
Previously it was the default-backend for the VICE K8s Ingress. Because of request routing issues, we need to convert it into a standalone service that users will get redirected to when they try to access a VICE app that isn't ready yet.

The idea is that the new default backend for the VICE ingress will redirect the user to this service with the app URL passed to it as a query parameter. The backend for this service will validate that the URL is valid (ensuring that the service can't be used to hammer an external site) and check to see if the app URL is accepting connections and HTTP requests. When both cases are true, the UI will set the window.location.href to the app's URL.

### CODE REVIEW

File: ui/src/components/App.js [hunk 1]
```diff
@@ -24,18 +25,8 @@ class App extends Component {
 
         <img src={loadingRocket} className="loading" alt="Loading rocket for an in-progress job" />
 
-        <div className="analysisMessage">Please wait while we launch your application.</div>
```
Comment (yeea) [inline ui/src/components/App.js]: BYE, DOTS

File: ui/src/components/loadingFeedback.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class LoadingDots extends Component {
+  render() {
+    return (
+      <div>
+        <div className="analysisMessage">Please wait while we launch your application.</div>
+
+        <div className="loading-dots">
```
Comment (yeea) [inline ui/src/components/loadingFeedback.js]: Oh, HI, DOTS

File: main.go [hunk 3]
```diff
@@ -677,20 +719,35 @@ func (c *CASProxy) isWebsocket(r *http.Request) bool {
...
+`
+
 func main() {
 	var (
 		err                      error
-		listenAddr               = flag.String("listen-addr", "0.0.0.0:8080", "The listen port number.")
+		cfg                      *viper.Viper
+		viceDomain               string
+		configPath               = flag.String("config", "/etc/iplant/de/jobservices.yml", "The path to the config file.")
+		listenAddr               = flag.String("listen-addr", "0.0.0.0:60000", "The listen port number.")
```
[the remark this answers is not in the document]
Comment (johnworth) [reply main.go]: I changed it to match the rest of the services, which use 60000 by default.

File: main.go [hunk 4]
```diff
@@ -677,20 +719,35 @@ func (c *CASProxy) isWebsocket(r *http.Request) bool {
 	return upgrade
 }
 
+const defaultConfig = `
```
[the remark this answers is not in the document]
Comment (johnworth) [reply main.go]: I don't think I do, the command-line default should be fine.

[the remark this answers is not in the document]
Comment (yeea) [reply main.go]: /Okay, then/

**Pull request merged**